src/main/kotlin/calculator/Operators.kt

```diff
@@ -0,0 +1,14 @@
+package calculator
+
+object Operators {
+    private val operators = mapOf(
+        Operator.PLUS.type to Operator.PLUS,
+        Operator.MINUS.type to Operator.MINUS,
+        Operator.MULTI.type to Operator.MULTI,
+        Operator.DIVIDE.type to Operator.DIVIDE
+    )
+
+    fun getOperator(stringOperate: String): Operator {
+        return operators[stringOperate] ?: throw IllegalArgumentException("+,-,/,* 연산만 가능합니다.")
+    }
```

Comment:
```suggestion
    fun getOperator(stringOperate: String): Operator = operators[stringOperate] ?: throw IllegalArgumentException("+,-,/,* 연산만 가능합니다.")
    
```

해당 경우에는 문(statement) 보다 간결성을 위하여 식(expression)으로 변경하는 게 좋지 않을까요?